api/pkg/auth/base.go

```diff
@@ -31,11 +33,22 @@ import (
 	auth "github.com/tektoncd/hub/api/pkg/auth/service"
 )
 
+// generate the random key if there is an error returns "SESSION_SECRET" as key
+func generateRandomKey(length int) string {
+	key := make([]byte, length)
+	_, err := rand.Read(key)
+	if err != nil {
+		return "SESSION_SECRET"
```

Comment:
@pratap0007 can you add `panic` of `fatal` on error response from generate random key. static key is not advised
@vdemeester @piyush-garg  WDYT?